kickstart/etc/smb.conf

```diff
@@ -39,3 +39,30 @@
    create mask = 0777
    directory mask = 0777
    writable = yes
+
+[omk]
+   comment = OpenMapKit Backups
+   path = /opt/data/backups/omk
```

Comment:
Consolidate these 3 shares into a single `[backup]` share exposing `path = /opt/data/backups`? That way we can add more backups via posm-admin or whatever and they'll "magically" show up shared.